oxide/lib.go

```diff
@@ -22,6 +25,12 @@ const TokenEnvVar = "OXIDE_TOKEN"
 // HostEnvVar is the environment variable that contains the host.
 const HostEnvVar = "OXIDE_HOST"
 
+var (
```

Comment:
Can these be constants instead?

Reply:
I can make the file names constants. Need to leave the directory as a private var because of `filepath.Join`. Done in 13a9c941b9ef981a01b0976eca5ee76747cc8fd4